_health-care/_js/components/military-service/AdditionalMilitaryInformationSection.jsx

```diff
@@ -12,91 +14,82 @@ class AdditionalMilitaryInformationSection extends React.Component {
           </div>
         </div>
 
         <div className="row">
           <div className="small-12 columns">
-            <input
-                type="checkbox"
-                name="veteran_is_former_pow"
-                id="veteran_is_former_pow"/>
-            <label htmlFor="veteran_is_former_pow">Are you a former prisoner of war?</label>
+            <ErrorableCheckbox
+                label="Are you a former prisoner of war?"
+                checked={this.props.data.isFormerPow}
+                onValueChange={(update) => {this.props.onStateChange('isFormerPow', update);}}/>
           </div>
         </div>
 
         <div className="row">
           <div className="small-12 columns">
-            <input
-                type="checkbox"
-                name="veteran_has_post_nov_11_1998_combat"
-                id="veteran_has_post_nov_11_1998_combat"/>
-            <label htmlFor="veteran_has_post_nov_11_1998_combat">Did you serve in combat theater of operations after November 11, 1998?</label>
+            <ErrorableCheckbox
+                label="Did you serve in combat theater of operations after November 11, 1998?"
+                checked={this.props.data.postNov111998Combat}
+                onValueChange={(update) => {this.props.onStateChange('postNov111998Combat', update);}}/>
           </div>
         </div>
 
         <div className="row">
           <div className="small-12 columns">
-            <input
-                type="checkbox"
-                name="veteran_is_disabled_in_line_of_duty"
-                id="veteran_is_disabled_in_line_of_duty"/>
-            <label htmlFor="veteran_is_disabled_in_line_of_duty">Were you discharged or retired from the military for a disability incurred in the line of duty?</label>
+            <ErrorableCheckbox
+                label="Were you discharged or retired from the military for a disability incurred in the line of duty?"
+                checked={this.props.data.disabledInLineOfDuty}
+                onValueChange={(update) => {this.props.onStateChange('disabledInLineOfDuty', update);}}/>
           </div>
         </div>
 
         <div className="row">
           <div className="small-12 columns">
-            <input
-                type="checkbox"
-                name="veteran_has_sw_asia_combat_aug_2_1990_to_nov_11_1998"
-                id="veteran_has_sw_asia_combat_aug_2_1990_to_nov_11_1998"/>
-            <label htmlFor="veteran_has_sw_asia_combat_aug_2_1990_to_nov_11_1998">Did you serve in SW Asia during the Gulf War between August 2, 1990 and Nov 11, 1998?</label>
+            <ErrorableCheckbox
+                label="Did you serve in SW Asia during the Gulf War between August 2, 1990 and Nov 11, 1998?"
+                checked={this.props.data.swAsiaCombatAug21990ToNov111998}
+                onValueChange={(update) => {this.props.onStateChange('swAsiaCombatAug21990ToNov111998', update);}}/>
```

Comment:
I know you were trying to be faithful to the original naming, but I wonder if we can shorten these? Something like `swAsiaCombat` instead? I think that should still provide the information we need to match this field to something in the xml block, while being slightly more readable. Just a thought.